docs/tutorials/standalone/Soil/layered_soil.jl

```diff
@@ -0,0 +1,171 @@
+vgn = reverse(parameter_data[5, :])
+residual_frac = reverse(parameter_data[2, :])
+porosity = reverse(parameter_data[3, :]);
+
+# Create fields corresponding to the parameter
+ν = SpaceVaryingInput(depth, porosity, soil_domain.space.subsurface)
+K_sat = SpaceVaryingInput(depth, ksat, soil_domain.space.subsurface)
+θ_r = SpaceVaryingInput(depth, residual_frac, soil_domain.space.subsurface);
+# The specific storativity is not something we have data on, so we
+# approximate it as being constant in depth, and create the parameter field directly:
+S_s = ClimaCore.Fields.zeros(soil_domain.space.subsurface) .+ 1e-3;
+
+# The retention model is a vanGenuchten model with α and n as a function of
+# depth, read from the data:
+hcm = SpaceVaryingInput(
+    depth,
+    (; α = vgα, n = vgn),
+    soil_domain.space.subsurface,
+    vanGenuchten{FT},
+);
+
+# The parameter struct:
+params = ClimaLand.Soil.RichardsParameters(;
+    ν = ν,
+    hydrology_cm = hcm,
+    K_sat = K_sat,
+    S_s = S_s,
+    θ_r = θ_r,
+);
+
+# From here on out, everything should look familiar if you've already gone
+# through the other soil tutorials.
+# Set Boundary conditions
+function top_flux_function(p, t)
+    # This is the observed value of Ksat at the top of the domain
+    # setting the flux to be -Ksat is approximating the top as saturated.
+    return -0.0001033
+end
+top_bc = ClimaLand.Soil.FluxBC(top_flux_function)
+bottom_bc = ClimaLand.Soil.FreeDrainage()
+boundary_fluxes = (; top = (water = top_bc,), bottom = (water = bottom_bc,))
+soil = Soil.RichardsModel{FT}(;
+    parameters = params,
+    domain = soil_domain,
+    boundary_conditions = boundary_fluxes,
+    sources = (),
+);
+
+# Initial the state vectors, and set initial conditions
+Y, p, cds = initialize(soil);
+
+# Initial conditions
+Y.soil.ϑ_l .= 0.0353; # read from Figure 4 of Huang et al. 
+
+# We also set the initial conditions of the auxiliary state here:
+set_initial_cache! = make_set_initial_cache(soil)
+set_initial_cache!(p, Y, t0);
+
+# Timestepping:
+stepper = CTS.ARS111()
+err = (FT == Float64) ? 1e-8 : 1e-4
+convergence_cond = CTS.MaximumError(err)
+conv_checker = CTS.ConvergenceChecker(norm_condition = convergence_cond)
+ode_algo = CTS.IMEXAlgorithm(
+    stepper,
+    CTS.NewtonsMethod(
+        max_iters = 50,
+        update_j = CTS.UpdateEvery(CTS.NewNewtonIteration),
+        convergence_checker = conv_checker,
+    ),
+)
+exp_tendency! = make_exp_tendency(soil)
+imp_tendency! = make_imp_tendency(soil)
+update_jacobian! = make_update_jacobian(soil)
+jac_kwargs =
+    (; jac_prototype = RichardsTridiagonalW(Y), Wfact = update_jacobian!)
+prob = SciMLBase.ODEProblem(
+    CTS.ClimaODEFunction(
+        T_exp! = exp_tendency!,
+        T_imp! = SciMLBase.ODEFunction(imp_tendency!; jac_kwargs...),
+        dss! = ClimaLand.dss!,
+    ),
+    Y,
+    (t0, tf),
+    p,
+)
+saveat = [0.0, 8.0, 16.0, 24.0, 32.0, 40.0, 60.0] .* 60 # chosen to compare with data in plots in paper
```

Comment:
It might be nice to also plot the data from the paper for comparison in the same plot like we do with the RRE Bonan comparison, but I don't think we have to :)

Reply:
I'd like to do that too! and in the soil paper we can because the people who collected the data would be co-authors. but I need to check with them if we can share it publicly. The data that is currently in this tutorial is in a table in their paper and so is public.